### DISCUSSION TITLE
i have no idea why everything is in code!!!

### DISCUSSION THREAD

**Discussion by jesss1ca** [community/community#38019] [Copilot Conversations]
I am so overwhelm3d by this!! I do not understand, why is everything in code , FREE code thats alll open, on my regular android cell phone? Every device i Own now has this on it... i do n ot work for some enterprise.... I have absolutely no idea how to make a "commit" , or what that even is at ALL......i need help...i just want regular use software, that I can understand.... I feel like nothing is private , and i dont know wehat to do

**Comment by jesss1ca** [comment]
any advice would be so appreciated. This has been a long ongoing thing, and I d0o not understand, Is it some sort of prank? Is it the person who purchased the phone for me, that set this up on my  phone, and how all these licenses that I do not need, got on my phone, my computer. I am so confused . I am not tech savvy and am clueless about this . :(

**Comment by david-engelmann** [comment]
Github is an Internet hosting service for software development and version control using Git. It provides the distributed version control of Git plus access control, bug tracking, software feature requests, task management, continuous integration, and wikis for every project. if you bought the phone used potentially they downloaded stuff from here. Attach screenshots might make it clearer what you are asking!

**Reply by jesss1ca** [reply]
Thank you for a fast reply! Everything is "as is"  "apache open source ". The phone was purchased new... It oriiginally was in the fine print on my previous phone (samsung A51 ) , and it overheated and the battery melted. Now , I had to start Using my child's  phone, after a factory hard reset (also new when purchased) , and I soon after began seeing the same fine print about this free software on it as well.   And, unsure why, but all my apps have automatic granted permissions  to all sensitive settings and data on ir, that I am unable to change. I am always seeing suggestions that I am using a "custom device" , and suggesting i am part of an organization of some kind, and that My data is modifiable and open to the public..

**Reply by david-engelmann** [reply]
Could you provide a screenshot of the fine print, additionally, can you find a .git or .github folder on it?
